example_grpc/goclient/src/katranc/main/main.go

```diff
@@ -38,14 +40,16 @@ var (
 		"Tcp service address. must be in format: <addr>:<port>")
 	udpService = flag.String("u", "",
 		"Udp service addr. must be in format: <addr>:<port>")
-	realServer     = flag.String("r", "", "Address of the real server")
-	realWeight     = flag.Int64("w", 1, "Weight (capacity) of real server")
-	showStats      = flag.Bool("s", false, "Show stats/counters")
-	showSumStats   = flag.Bool("sum", false, "Show summary stats")
-	showLruStats   = flag.Bool("lru", false, "Show LRU related stats")
-	showIcmpStats  = flag.Bool("icmp", false, "Show ICMP 'packet too big' related stats")
-	listServices   = flag.Bool("l", false, "List configured services")
-	vipChangeFlags = flag.String("vf", "",
+	realServer      = flag.String("r", "", "Address of the real server")
+	realWeight      = flag.Int64("w", 1, "Weight (capacity) of real server")
+	showStats       = flag.Bool("s", false, "Show stats/counters")
+	showSumStats    = flag.Bool("sum", false, "Show summary stats")
+	showLruStats    = flag.Bool("lru", false, "Show LRU related stats")
+	showIcmpStats   = flag.Bool("icmp", false, "Show ICMP 'packet too big' related stats")
+	listServices    = flag.Bool("l", false, "List configured services")
+	dumpServices    = flag.Bool("L", false, "Dump List configured services as json file")
+	servicesJsoPath = flag.String("json", "services.json", "List configured services")
```

Comment:
servicesJsonPath.
Also, can you update the description from "List configured services" to reflect what it does?

Reply:
fixed